content/maxwell2_dc/fields_from_grounded_sources_dcr/electrostatic_sphere.rst

```diff
@@ -10,9 +10,9 @@ electrostatic field. This scenario gives us a setting to examine aspects of
 the DC resistivity experiment, including the behavior of electric potentials,
 electric fields, current density and the build up of charges at interfaces.
 This work follows the derivation in :cite:`ward1988` and is supported by apps developed in
-a `Jupyter Notebook`_.
+a `binder`_.
 
-.. _Jupyter Notebook: https://github.com/ubcgif/em/blob/master/examples/sphere/ElectrostaticSphere.ipynb
+.. _binder:  http://mybinder.org/repo/ubcgif/em_examples/notebooks/Maxwell2_DC/electrostatic_sphere/ElectrostaticSphere_example.ipynb
```

Comment:
just fyi, if you want the badge [![Binder](http://mybinder.org/badge.svg)](http://mybinder.org/repo/ubcgif/em_examples/notebooks/Maxwell2_DC/electrostatic_sphere/ElectrostaticSphere_example.ipynb), you can use
 ```
.. image:: http://mybinder.org/badge.svg 
    :target: http://mybinder.org/repo/ubcgif/em_examples/notebooks/Maxwell2_DC/electrostatic_sphere/ElectrostaticSphere_example.ipynb
```
just like you would for any other figure